src/mesido/workflows/grow_workflow.py

```diff
@@ -205,27 +203,38 @@ def read(self):
         # Error checking:
         # - installed capacity/power of a heating/cooling demand is sufficient for the specified
         #   demand profile
-        is_error = False
-        for error_type, errors in self._asset_potential_errors.items():
-            if error_type in ["heat_demand.power", "cold_demand.power"]:
-                if len(errors) > 0:
-                    for asset_name in errors:
-                        logger.error(self._asset_potential_errors[error_type][asset_name])
-                    logger.error(
-                        "Asset insufficient installed capacity: please increase the"
-                        " installed power or reduce the demand profile peak value of the demand(s)"
-                        " listed."
+        # - asset used for a heating demand
+        potential_error_type = [
```

Comment:
Great functionality! Can we maybe make a function of this and put it in the utils? I think we will reuse these checks in other/future workflows and by putting it in a separate function in utils we can minimize the code repetition.

Reply:
@JimRojer thanks for the feedback. The code has been updated so that this is all replaced by 1 function now. @FJanssen-TNO  will review it